package.json

```diff
@@ -2,10 +2,16 @@
   "name": "kata-07-rejoj-checador-equipo-02-k07",
   "private": true,
   "dependencies": {
+    "@rails/actioncable": "^6.0.0",
+    "@rails/activestorage": "^6.0.0",
     "@rails/ujs": "^6.0.0",
+    "@rails/webpacker": "5.2.1",
+    "bootstrap": "^5.0.0-beta3",
     "turbolinks": "^5.2.0",
-    "@rails/activestorage": "^6.0.0",
-    "@rails/actioncable": "^6.0.0"
+    "yarn": "^1.22.10"
```

Comment:
is this necessary?

Reply:
I'm not sure, it was generated automatically:c

Reply:
I talked to Anxhe and she said it isn't necessary.